src/pages/errorpages/Forbidden.tsx

```diff
@@ -1,20 +1,71 @@
-import { Box, Link as MuiLink, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import LockOutlineIcon from '@mui/icons-material/LockOutline';
+import LoginIcon from '@mui/icons-material/Login';
+import { Box, Button, SxProps, Typography } from '@mui/material';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 import { Link } from 'react-router';
+import { RootState } from '../../redux/store';
+import { LocalStorageKey } from '../../utils/constants';
+import { dataTestId } from '../../utils/dataTestIds';
+import { useAuthentication } from '../../utils/hooks/useAuthentication';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 
+const buttonProps: SxProps = { textTransform: 'none', mt: '1rem', width: 'fit-content' };
+
 export const Forbidden = () => {
   const { t } = useTranslation();
+  const { handleLogin } = useAuthentication();
+  const user = useSelector((store: RootState) => store.user);
 
   return (
-    <Box data-testid="forbidden" sx={{ mt: '4rem' }}>
-      <Typography variant="h2" component="h1" sx={{ mb: '1rem' }}>
-        {t('authorization.forbidden')}
-      </Typography>
-      <Typography sx={{ mb: '1rem' }}>{t('authorization.forbidden_description')}</Typography>
-      <MuiLink component={Link} to={UrlPathTemplate.Root}>
-        <Typography>{t('authorization.back_to_home')}</Typography>
-      </MuiLink>
+    <Box data-testid="forbidden" sx={{ my: '4rem', display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+      <div>
+        <span style={{ display: 'flex', gap: '0.5rem' }}>
+          <LockOutlineIcon />
+          <Typography gutterBottom variant="h1">
+            {t('authorization.forbidden')}
+          </Typography>
+        </span>
+
+        {!user ? (
```

Comment:
Pirk, men IMO er det som regel mer lesbart å å `!` i else, og heller ha sjekken på den sanne verdien. Bare for å unngå den lille ekstra overheaden med at du må negere verdien selv når man leser. Ikke noen koblisert sjekk her uansett da, men noe det kan være greit å tenke litt over

Reply:
Forsåvidt enig der. Tenkte nok mer "flyt"-messig her